cmd/helm/delete.go

```diff
@@ -18,12 +18,13 @@ deleting them.
 var deleteDryRun bool
 
 var deleteCommand = &cobra.Command{
-	Use:        "delete [flags] RELEASE_NAME",
-	Aliases:    []string{"del"},
-	SuggestFor: []string{"remove", "rm"},
-	Short:      "Given a release name, delete the release from Kubernetes",
-	Long:       deleteDesc,
-	RunE:       delRelease,
+	Use:               "delete [flags] RELEASE_NAME",
+	Aliases:           []string{"del"},
+	SuggestFor:        []string{"remove", "rm"},
+	Short:             "Given a release name, delete the release from Kubernetes",
+	Long:              deleteDesc,
+	RunE:              delRelease,
+	PersistentPreRunE: setupConnection,
```

Comment:
Asking from sheer ignorance... why use `PersistentPreRunE` instead of `PreRunE`?